templates/includes/header.html

```diff
@@ -3,14 +3,14 @@
     <div class="p-navigation__banner">
       <div class="p-navigation__logo">
         <a href="/" class="p-navigation__link">
-          <img src="https://assets.ubuntu.com/v1/e1bdea10-MAAS+White.svg" alt="" width="100" height="48">
+          <img src="https://assets.ubuntu.com/v1/e1bdea10-MAAS+White.svg" alt="MAAS" width="100" height="48">
```

Comment:
If this is the only text in the link then should it read "Go to homepage for maas.io"

Reply:
Good point! "Homepage" is probably more appropriate than just "MAAS". Updated.